bucket/carotdav.json

```
@@ -0,0 +1,35 @@
+{
+    "homepage": "http://rei.to/carotdav_en.html",
+    "description": "A simple WebDAV / FTP / SFTP / Online Storages client for Windows OS",
+    "license": {
+        "identifier": "Freeware",
+        "url": "http://rei.to/carotdav_en.html#license"
+    },
+    "version": "1.15.5",
+    "url": "http://rei.to/CarotDAV1.15.5.portable.zip",
+    "hash": "21b0b68ec45223c747d67b0e21ed87e8c195d52e5fc651b30b631a2f328bcd01",
```

```suggestion
    "hash": "sha1:45a3859de58bde9c12fd4792fa54d48fb0a30184",
```

`hash` gotten by `checkver`